pinot-controller/src/test/java/org/apache/pinot/controller/validation/ResourceUtilizationManagerTest.java

```diff
@@ -28,58 +30,88 @@
 public class ResourceUtilizationManagerTest {
 
   private DiskUtilizationChecker _diskUtilizationChecker;
+  private List<UtilizationChecker> _utilizationCheckers;
   private ControllerConf _controllerConf;
   private ResourceUtilizationManager _resourceUtilizationManager;
   private final String _testTable = "myTable_OFFLINE";
 
   @BeforeMethod
   public void setUp() {
     _diskUtilizationChecker = Mockito.mock(DiskUtilizationChecker.class);
+    _utilizationCheckers = Collections.singletonList(_diskUtilizationChecker);
```

Comment:
(minor) Use `List.of(_diskUtilizationChecker)`

Reply:
done